### PR TITLE
Update CZ.cup frequencies for 2022

### PR DESCRIPTION
**Pull request by DanD222**
Differences identified in:
https://aim.rlp.cz/vfrmanual/actual/lkbrez_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkco_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkct_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkdk_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkhrad_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkjiri_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkkune_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkkv_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkleto_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lklt_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkmilo_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkmr_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkpd_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkpisk_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkpr_text_en.html
https://aim.rlp.cz/vfrmanual/actual/lkto_text_en.html

### CODE REVIEW

File: data/content/waypoint/country/CZ.cup [hunk 1]
```diff
@@ -222,7 +222,7 @@ name,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc
 "HRANICE","LKHN",CZ,4932.767N,01742.267E,243.0m,2,50,735.0m,123.510,"Hranice, ARP"
 "HORY-P","LKHORY",CZ,5013.022N,01246.614E,502.0m,2,70,450.0m,125.830,"Hory, ULL Private"
 "HOLESOV-X","LKHO-X",CZ,4918.867N,01734.133E,231.0m,3,0,0.0m,,"Holesov, ARP CLSD"
-"HRADCANY U MIMONE-P","LKHR",CZ,5037.167N,01443.967E,278.0m,5,90,2700.0m,134.815,"Hradcany u Mimone, ULL Private"
+"HRADCANY U MIMONE-P","LKHRAD”,CZ,5037.167N,01443.967E,278.0m,5,90,2700.0m,134.815,"Hradcany u Mimone, ULL Private"
```
Comment (lordfolken) [inline data/content/waypoint/country/CZ.cup:225]: "LKHRAD” <- the trailing " is a different format than the leading one.

File: data/content/waypoint/country/CZ.cup [hunk 2]
```diff
@@ -300,19 +300,19 @@ name,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc
 "PANOSI UJEZD","LKPANO",CZ,5002.320N,01341.819E,475.0m,2,90,420.0m,125.830,"Panosi Ujezd, ULL"
 "VELKE PAVLOVICE","LKPAVL",CZ,4852.835N,01649.492E,172.0m,2,160,605.0m,130.555,"Velke Pavlovice, ULL"
 "PANENSKY TYNEC","LKPC",CZ,5018.383N,01356.050E,368.0m,5,90,2505.0m,118.580,"Panensky Tynec, ARP"
-"PARDUBICE MTWR","LKPD",CZ,5000.807N,01544.314E,226.0m,5,90,2500.0m,120.155,"Pardubice, ARP, MTWR 120.155, APP 127.655"
+"PARDUBICE MTWR","LKPD",CZ,5000.807N,01544.314E,226.0m,5,90,2500.0m,120.155,"Pardubice, ARP, MTWR 120.155, APP 128.365"
 "PENCIN","LKPENC",CZ,4934.100N,01659.933E,371.0m,2,150,340.0m,125.830,"Pencin, ULL"
 "PRIBYSLAV","LKPI",CZ,4934.833N,01545.767E,531.0m,2,80,765.0m,125.610,"Pribyslav, ARP"
 "PROSTEJOV","LKPJ",CZ,4926.867N,01708.033E,214.0m,2,120,1000.0m,126.905,"Prostejov, ARP"
-"PISEK","LKPK",CZ,4920.367N,01406.833E,412.0m,5,160,600.0m,125.830,"Pisek, ULL"
+"PISEK","LKPISK",CZ,4920.367N,01406.833E,412.0m,5,160,600.0m,119,415,"Pisek, ULL"
```
Comment (lordfolken) [inline data/content/waypoint/country/CZ.cup:307]: The frequency field contains a , instead of a .

**Review by lordfolken — CHANGES_REQUESTED** [review]
Some formatting errors, which is why the check fails.

**Comment by DanD222** [comment]
@lordfolken Should be OK to merge now?

**Comment by lordfolken** [comment]
The check is still failing.

**Comment by DanD222** [comment]
Thanks.. spotted the other quotation mark, missed the line ending :(

**Pull request merged**